<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buy_NewCar</class>
 <widget class="QMainWindow" name="buy_NewCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="refresh">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Company</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>361</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>210</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>7</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Insert New Company</string>
         </property>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>180</y>
           <width>191</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>7</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Find the company ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_comp">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Manager</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>361</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>NationalID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>7</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Insert New Manager</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_ans">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>140</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>190</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Manager ID is :</string>
         </property>
        </widget>
        <widget class="QLabel" name="mgrid">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>190</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>New Car</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Car Information</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_color">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>70</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_engine">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>70</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_factory">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_bprice">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_model">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>110</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>110</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Model*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Buy Price*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Factory*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>70</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Engine Number*</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_crcar">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>110</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>*Essential fields</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save_car">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>150</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>150</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Car ID is:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_car">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>150</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="buy_new_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>588</y>
         <width>874</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>BUY</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="buy_new">
      <property name="text">
       <string>BUY</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
